holidays/countries/dr_congo.py

```diff
@@ -0,0 +1,137 @@
+        * [Ordonnance n° 14/010](https://web.archive.org/web/20230419184344/http://leganet.cd/Legislation/Divers/Ordonnance.14.10.14.mai.2014.htm)
+        * [Ordonnance n° 23-042](https://web.archive.org/web/20250113230411/http://www.droitcongolais.info/files/143.03.23_Ordonnance-du-30-mars-2023_jours-feries.pdf)
+        * [Loi n° 009-2002](https://web.archive.org/web/20250104233847/https://www.leganet.cd/Legislation/Droit%20administratif/Urbanismevoiries/Div/L.009.05.08.2002.htm)
+    """
+
+    country = "CD"
+    default_language = "fr"
+    # %s (observed).
+    observed_label = tr("%s (observé)")
+    supported_languages = ("en_US", "fr")
+    # Ordonnance n° 79-154.
+    start_year = 1980
+
+    def __init__(self, *args, **kwargs) -> None:
+        ChristianHolidays.__init__(self)
+        InternationalHolidays.__init__(self)
+        kwargs.setdefault("observed_rule", SUN_TO_PREV_SAT)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # New Year's Day.
+        self._add_observed(self._add_new_years_day(tr("Nouvel an")))
+
+        # Established on May 10th, 2014 via Ordonnance n° 14/010.
+        if self._year >= 2015:
+            # Martyrs' Day.
+            self._add_observed(self._add_holiday_jan_4(tr("Martyrs de l'indépendance")))
+
+            self._add_observed(
+                # National Hero Laurent Désiré Kabila Day.
+                self._add_holiday_jan_16(tr("Journée du héros national Laurent Désiré Kabila"))
+            )
+
+            self._add_observed(
+                # National Hero Patrice Emery Lumumba Day.
+                self._add_holiday_jan_17(tr("Journée du héros national Patrice Emery Lumumba"))
+            )
+
+        # Established on March 30th, 2023 via Ordonnance n° 23-042.
+        if self._year >= 2023:
+            self._add_observed(
+                self._add_holiday_apr_6(
+                    # Day of the Struggle of Simon Kimbangu and African Consciousness.
+                    tr("Journée du combat de Simon Kimbangu et de la conscience africaine")
+                )
+            )
+
+        # Labor Day.
+        self._add_observed(self._add_labor_day(tr("Fête du travail")))
+
+        # Removed on May 10th, 2014 via Ordonnance n° 14/010.
+        if self._year <= 2014:
+            self._add_observed(
+                self._add_holiday_may_20(
+                    # Anniversary of the Popular Movement of the Revolution.
+                    tr("Anniversaire du Mouvement populaire de la révolution")
+                )
+            )
+
+            self._add_observed(
+                self._add_holiday_jun_24(
+                    # Anniversary of the New Revolutionary Constitution.
+                    tr("Anniversaire de la nouvelle Constitution révolutionnaire")
+                )
+            )
+
+        # Renamed on May 10th, 2014 via Ordonnance n° 14/010.
+        if self._year >= 2015:
```

Comment:
```suggestion
        if self._year >= 2014:
```
Ordonnance - May 10, holiday - May 17.
And this holiday should be placed after Labor Day (chronological order).